tests/ui/stable-mir-print/basic_function.stdout

```diff
@@ -0,0 +1,39 @@
+// WARNING: This is highly experimental output it's intended for stable-mir developers only.
+// If you find a bug or want to improve the output open a issue at https://github.com/rust-lang/project-stable-mir.
+fn foo(_0: i32) -> i32 {
+    let mut _0: (i32, bool);
+
+    bb0: {
+        _2 = 1 Add const 1_i32
+    }
+    bb1: {
+        _0 = move _2
+    }
+}
+fn bar(_0: &mut std::vec::Vec) -> std::vec::Vec {
+    let mut _0: std::vec::Vec;
+    let mut _1: &std::vec::Vec;
+    let _2: ();
+    let mut _3: &mut std::vec::Vec;
+
+    bb0: {
+        _3 = refShared1
+    }
+    bb1: {
+        _5 = refMut {
+    kind: TwoPhaseBorrow,
+}2
```

Comment:
That's still messed up. Since this PR is about tweaks, could you fix this here?

Reply:
I think we might want to render this like `&mut (two-phase) ....`

Reply:
🤔  that should look like this
```
_5 = &mut _2;
```

Reply:
I think there was an open issue about whether or not to print two-phase borrows as part of the operator, but yeah, if we're trying to mirror `-Zunpretty=mir` then we could just say `&mut` and ignore the two-phase bool.

Reply:
We actually haven't decided on how much we want this to look like `-Zunpretty=mir` but this one doesn't provide any information and it looks totally broken.